packages/component/src/types/external/OrgSchema/EntryPoint.ts

```diff
@@ -0,0 +1,21 @@
+import { isThingOf, type Thing } from './Thing';
+
+/**
+ * An entry point, within some Web-based protocol.
+ *
+ * This is partial implementation of https://schema.org/EntryPoint.
+ *
+ * @see https://schema.org/EntryPoint
+ */
+export type EntryPoint = Thing<'EntryPoint'> &
+  Readonly<{
+    /** Indicates the current disposition of the Action. */
+    actionPlatform?: string | undefined | URL;
+
+    /** An url template (RFC6570) that will be used to construct the target of the execution of the action. */
+    urlTemplate?: string | undefined;
```

Comment:
In the `tsconfig.json` of my playground, I enabled something called `exactOptionalPropertyTypes` (via `@tsconfig/strictest`) and it enforce `undefined` because "optional !== `undefined`". 😅

I am going to keep the `| undefined` as I plan to move forward to strictest.